PopPUNK/assign.py

```diff
@@ -132,7 +127,7 @@ def assign_query(dbFuncs,
                                     codon_phased = codon_phased,
                                     calc_random = False)
     else:
-        qNames = sketch_to_hdf5(sketch, output)
+        qNames = sketch_to_hdf5(json_sketch, output)
```

Comment:
It probably makes sense to check that `json_sketch` and `output` have been defined if `web == True` (as on e.g. line 85)